DFC.ServiceTaxonomy.ContentApproval/Startup.cs

```diff
@@ -4,20 +4,27 @@
 using DFC.ServiceTaxonomy.ContentApproval.Indexes;
 using DFC.ServiceTaxonomy.ContentApproval.Models;
 using DFC.ServiceTaxonomy.ContentApproval.Permissions;
-using DFC.ServiceTaxonomy.ContentApproval.Shapes;
 using DFC.ServiceTaxonomy.ContentApproval.Services;
+using DFC.ServiceTaxonomy.ContentApproval.Shapes;
+//using DFC.ServiceTaxonomy.ContentApproval.Services;
```

Comment:
Should we leave commented out? Are they to be restored soon?

Reply:
Probably should be removed now. Think this must have been missed after the refactor of the Dashboard querying.
Rather than do this here i'll fix on the dev branch so that is will be cleared for future releases.